packages/fiori3/src/components/AnalyticalTable/index.tsx

```diff
@@ -0,0 +1,266 @@
+import { VerticalAlign } from '../../enums/VerticalAlign';
+import { TextAlign } from '../../enums/TextAlign';
+import React, { Component, CSSProperties, ReactNode, ReactText } from 'react';
+import ReactTable from 'react-table';
+import '../../../../../node_modules/react-table/react-table.css';
```

Comment:
```suggestion
import 'react-table/react-table.css';
```
should be sufficient. Is there also a minified css file available?